test_requirements.txt

```diff
@@ -6,3 +6,4 @@ thriftpy2
 six
 netifaces
 sphinx_autodoc_typehints
+sphinx_rtd_theme
```

Comment:
This seems to contradict https://github.com/xnd-project/rbc/pull/440 that replaces sphinx_rtd_theme with pydata-sphinx-theme.

Reply:
fixed